python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/aaa-accounting.j2

```diff
@@ -34,8 +34,20 @@
 {%     if aaa_accounting.system.default.type is arista.avd.defined and aaa_accounting.system.default.group is arista.avd.defined  %}
 | System - Default | - | {{ aaa_accounting.system.default.type }} | {{ aaa_accounting.system.default.group }} | - |
 {%     endif %}
-{%     if aaa_accounting.dot1x.default.type is arista.avd.defined and aaa_accounting.dot1x.default.group is arista.avd.defined %}
-| Dot1x - Default  | - | {{ aaa_accounting.dot1x.default.type }} | {{ aaa_accounting.dot1x.default.group }} | - |
+{%     if aaa_accounting.dot1x.default.type is arista.avd.defined and aaa_accounting.dot1x.default.methods is arista.avd.defined %}
+{%         set methods_list = [] %}
+{%         set ns = namespace(method_logging = false) %}
```

Comment:
Do not use the sort form in variable name. Please update with logging_namespace or logging_name_space

Reply:
Fixed.